runtime/interpreter/interpreter.go

```diff
@@ -2618,65 +2618,15 @@ func init() {
 		BaseActivation,
 		"DictionaryType",
 		NewUnmeteredHostFunctionValue(
-			func(invocation Invocation) Value {
```

Comment:
Just extracted as global functions to make the outer function shorter / easier to reason about